kalite/settings/base.py

```diff
@@ -248,9 +255,18 @@
         "OPTIONS": {
             "timeout": 60,
         },
+    },
+    "topic_tools": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": TT_DATABASE_PATH,
+        "OPTIONS": {
+            "timeout": 60,
```

Comment:
Dunno, I just figured it seemed reasonable to have the same options as the other database.